### ISSUE TITLE
macOS: No such file or directory (os error 2)

### ISSUE THREAD

**Issue by noestreich** [Fredolx/open-tv#61]
If i try to open the attached single entry playlist, i get the error: "**No such file or directory (os error 2)**"

<img width="912" alt="Bildschirmfoto 2024-10-16 um 11 37 16" src="https://github.com/user-attachments/assets/6c5df787-9b52-4d11-b9e1-ac5b71a5d5cd">

My Playlist-File: (Ideally i like to open the files from the Free-TV-Project [https://github.com/Free-TV/IPTV](https://github.com/Free-TV/IPTV) )
```
#EXTM3U
#EXTINF:-1 tvg-name="Das Erste Ⓖ" tvg-logo="https://upload.wikimedia.org/wikipedia/commons/thumb/c/ca/Das_Erste_2014.svg/640px-Das_Erste_2014.svg.png" tvg-id="DasErste.de" group-title="Germany",Das Erste Ⓖ
https://mcdn.daserste.de/daserste/de/master.m3u8

```

**Comment by Fredolx** [comment]
does the URL work? I will test this on MacOS today. Thank you

**Comment by noestreich** [comment]
Same arrive if I tried this link for the German playlist: https://raw.githubusercontent.com/Free-TV/IPTV/master/playlists/playlist_germany.m3u8

**Comment by noestreich** [comment]
As requested: 
https://github.com/user-attachments/assets/5c320fe8-a924-4d6f-945b-0a7309a28447

**Comment by Fredolx** [comment]
I better understand now,

Do you have Open TV 1.0.2 installed?
And mpv, yt-dlp installed from brew or MacPorts?

**Comment by Fredolx** [comment]
The app depends on mpv, ffmpeg and yt-dlp. On MacOS, you must use Brew or MacPorts to install those dependencies.

```
brew install mpv yt-dlp ffmpeg
```

**Comment by noestreich** [comment]
All of them ar already installed.

**Comment by noestreich** [comment]
Progress. MVP was missing. Is installed now. New Error-message: "Error parsing option cache no (option not found)Setting commandline option --cache no= failed."

**Comment by Fredolx** [comment]
Hi @noestreich
You are right about the caching, there is currently a bug with it. I will be fixing it. In the meantime, please put stream caching on to avoid issues.

<img width="778" alt="Screenshot 2024-10-17 at 10 40 23 AM" src="https://github.com/user-attachments/assets/9bb0f723-1566-4446-9e43-4d56d824d197">

**Comment by Fredolx** [comment]
The bug has been fixed with 1.0.3
